conda/cli/find_commands.py

```diff
@@ -25,7 +25,7 @@ def find_executable(executable, include_others=True):
 
     for dir_path in dir_paths:
         if sys.platform == 'win32':
-            for ext in  '.exe', '.bat', '':
+            for ext in {'.exe', '.bat', ''}:
```

Comment:
Any reason not to just keep this a tuple?

    for ext in ('.exe', '.bat', ''):